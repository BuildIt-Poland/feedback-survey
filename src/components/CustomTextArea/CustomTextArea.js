import React from 'react';
import { TextArea } from 'grommet';

const CustomTextArea = ({ ...rest }) => <TextArea {...rest} />;

export default CustomTextArea;
