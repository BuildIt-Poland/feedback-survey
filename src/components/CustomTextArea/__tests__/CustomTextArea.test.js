import React from 'react';
import { create } from 'react-test-renderer';

import CustomTextArea from '..';

describe('COMPONENT - CustomTextArea', () => {
  it('renders correctly', () => {
    const component = create(<CustomTextArea />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
