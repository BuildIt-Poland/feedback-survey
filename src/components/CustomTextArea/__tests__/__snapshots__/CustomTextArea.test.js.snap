// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - CustomTextArea renders correctly 1`] = `
<textarea
  className="StyledTextArea-sc-17i3mwp-0 fUSzal"
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
/>
`;
